dal/daltest/helpers.go

```diff
@@ -75,53 +76,53 @@ func PopulateSignRequests(t *testing.T, db dal.DB) []dal.IkaSignRequest {
 	}
 
 	for _, request := range requests {
-		err := db.InsertIkaSignRequest(request)
+		err := db.InsertIkaSignRequest(ctx, request)
 		assert.NilError(t, err)
 	}
 
 	return requests
 }
 
 // PopulateIkaTxs inserts a set of predefined IkaTxs into the database.
-func PopulateIkaTxs(t *testing.T, db dal.DB) []dal.IkaTx {
+func PopulateIkaTxs(ctx context.Context, t *testing.T, db dal.DB) []dal.IkaTx {
 	t.Helper()
 
 	ikaTxs := []dal.IkaTx{
-		{SrID: 1, Status: dal.Success, IkaTxID: "ika_tx_1", Timestamp: time.Now().Unix(), Note: ""},
-		{SrID: 2, Status: dal.Success, IkaTxID: "ika_tx_2", Timestamp: time.Now().Unix(), Note: ""},
-		{SrID: 3, Status: dal.Failed, IkaTxID: "ika_tx_3", Timestamp: time.Now().Unix(), Note: "some error"},
+		{SrID: 1, Status: int64(dal.Success), IkaTxID: "ika_tx_1", Timestamp: time.Now().Unix()},
```

Comment:
maybe there is a way to set the types for `sqlc`? Otherwise, let's change type of `dal.Success` (and related enums) to `int64`

Reply:
I found it - there is a way: https://docs.sqlc.dev/en/stable/howto/overrides.html#the-overrides-list

Reply:
Done